### PR TITLE
Use forked version of htmlproofer

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: .docker/entrypoint/Gemfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ end
 gem 'docopt'
 # Neccessary to prevent Jekyll errors. See https://github.com/github/personal-website/issues/166
 gem 'faraday', '~> 1.0.1'
-gem 'html-proofer'
+gem 'html-proofer', git: 'https://github.com/NullableInt/html-proofer', branch: 'feature/reduce_memory_ussage'
```
Comment (payex-dev) [inline .docker/entrypoint/Gemfile:19]: Codacy found an issue: [Line is too long. [110/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/Gemfile [hunk 2]
```diff
@@ -16,7 +15,7 @@ end
...
```
Comment (payex-dev) [inline .docker/entrypoint/Gemfile:18]: Codacy found an issue: [Line is too long. [110/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/Gemfile [hunk 3]
```diff
@@ -7,16 +7,16 @@ group :jekyll_plugins do
...
   gem 'kramdown-plantuml'
   gem 'rouge'
 end
 
+gem 'concurrent-ruby'
 gem 'docopt'
 # Neccessary to prevent Jekyll errors. See https://github.com/github/personal-website/issues/166
 gem 'faraday', '~> 1.0.1'
-gem 'html-proofer'
+gem 'html-proofer', git: 'https://github.com/NullableInt/html-proofer', branch: 'feature/cache_local_links'
```
Comment (payex-dev) [inline .docker/entrypoint/Gemfile:19]: Codacy found an issue: [Line is too long. [107/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/lib/commands/verifier.rb [hunk 4]
```diff
@@ -48,34 +63,27 @@ def options(ignore_urls = nil)
...
-              }
+            check_unrendered_link: true,
+            parallel: { in_processes: Concurrent.processor_count },
+            typheous: {
+              verbose: @context.verbose?
+            },
+            verbose: @context.verbose?,
+            timeframe: "1h",
+            cache: {
+              timeframe: "1h"
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/verifier.rb:86]: Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/lib/commands/verifier.rb [hunk 5]
```diff
@@ -48,34 +63,27 @@ def options(ignore_urls = nil)
...
-              values: {
-                token: auth_token
-              }
+            check_unrendered_link: true,
+            parallel: { in_processes: Concurrent.processor_count },
+            typheous: {
+              verbose: @context.verbose?
+            },
+            verbose: @context.verbose?,
+            timeframe: "1h",
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/verifier.rb:84]: Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/lib/commands/verifier.rb [hunk 6]
```diff
@@ -48,34 +63,29 @@ def options(ignore_urls = nil)
...
-              }
+            check_unrendered_link: true,
+            report_mismatched_tags: true,
+            parallel: { in_processes: Concurrent.processor_count },
+            typheous: {
+              verbose: @context.verbose?
+            },
+            verbose: @context.verbose?,
+            cache: {
+              timeframe: "1h"
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/verifier.rb:88]: Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/Gemfile (hunk 3 above)
Comment (payex-dev) [inline .docker/entrypoint/Gemfile:19]: Codacy found an issue: [Line is too long. [107/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/lib/commands/verifier.rb [hunk 7]
```diff
@@ -48,35 +63,31 @@ def options(ignore_urls = nil)
...
 
           opts
         end
 
         def default_options
-          opts = {
+          {
             assume_extension: true,
             check_html: true,
+            #check_favicon: true,
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/verifier.rb:76]: Codacy found an issue: [Missing space after `#`.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/Gemfile (hunk 3 above)
Comment (payex-dev) [inline .docker/entrypoint/Gemfile:19]: Codacy found an issue: [Line is too long. [107/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/lib/commands/verifier.rb [hunk 8]
```diff
@@ -48,35 +63,31 @@ def options(ignore_urls = nil)
...
+            report_mismatched_tags: true,
+            #parallel: { in_processes: Concurrent.processor_count },
+            typheous: {
+              verbose: @context.verbose?
+            },
+            verbose: @context.verbose?,
+            cache: {
+              timeframe: '1h'
+            },
+            checks_to_ignore: ['opengraphcheck', 'scriptcheck', 'chercker']
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/verifier.rb:90]: Codacy found an issue: [Use `%w` or `%W` for an array of words.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/lib/commands/verifier.rb [hunk 9]
```diff
@@ -48,35 +63,31 @@ def options(ignore_urls = nil)
...
-            'github.com' => {
-              type: :header,
-              template: 'Bearer %token%',
-              values: {
-                token: auth_token
-              }
-            }
+            check_unrendered_link: true,
+            report_mismatched_tags: true,
+            #parallel: { in_processes: Concurrent.processor_count },
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/verifier.rb:82]: Codacy found an issue: [Missing space after `#`.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/lib/commands/verifier.rb [hunk 10]
```diff
@@ -48,35 +63,31 @@ def options(ignore_urls = nil)
...
           opts
         end
 
         def default_options
-          opts = {
+          {
             assume_extension: true,
             check_html: true,
+            #check_favicon: true,
+            #check_opengraph: true,
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/verifier.rb:77]: Codacy found an issue: [Missing space after `#`.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/Gemfile (hunk 3 above)
Comment (payex-dev) [inline .docker/entrypoint/Gemfile:19]: Codacy found an issue: [Line is too long. [107/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/lib/commands/verifier.rb [hunk 11]
```diff
@@ -48,35 +63,31 @@ def options(ignore_urls = nil)
...
+            report_mismatched_tags: true,
+            parallel: { in_processes: Concurrent.processor_count },
+            typheous: {
+              verbose: @context.verbose?
+            },
+            verbose: @context.verbose?,
+            cache: {
+              timeframe: '1h'
+            },
+            checks_to_ignore: ['opengraphcheck', 'scriptcheck', 'chercker']
```
Comment (payex-dev) [inline .docker/entrypoint/lib/commands/verifier.rb:90]: Codacy found an issue: [Use `%w` or `%W` for an array of words.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

File: .docker/entrypoint/Gemfile (hunk 3 above)
Comment (payex-dev) [inline .docker/entrypoint/Gemfile:19]: Codacy found an issue: [Line is too long. [107/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)

**Comment by asbjornu** [comment]
I suppose this can be closed now @NullableInt?

**Pull request closed without merging**